### PR TITLE
Smear QCD and Store events

### PR DESCRIPTION
**Pull request by mkilpatr**
Here is the code to smear the file and creates new events and stores them in new files. There is a script that will hadd all the files together since there is a max number of files that haddnano.py can do.

### CODE REVIEW

**Comment by mkilpatr** [comment]
Forgot to add the output file from the process file to the hadd script. Problem is fixed

File: python/processors/haddnano.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#!/bin/env python
```
Comment (zhenbinwu) [inline python/processors/haddnano.py]: Is this haddnano.py the same as the one from the NanoAOD-tools?

**Comment by mkilpatr** [comment]
It is but I’m actually using a new one now that can hadd more files
together. I’ll do a new PR.

**Comment by zhenbinwu** [comment]
Is this PR finalized? Once the code is ready, should we merge it into the master?

**Comment by mkilpatr** [comment]
I'm close to getting the bootstrapping to work and then it should be good
to go.

**Comment by mkilpatr** [comment]
Able to save the smear events to a single file for any number of smears

**Comment by mkilpatr** [comment]
New jetresskim file used to save only the needed values in a flat tree to be used on the already existing smearing code.

**Comment by mkilpatr** [comment]
There are now separate modules for the response files and qcd smearing files. I have added the nanoaod framework to allow QCD Smearing, response calculation, and base calculation in the same framework. The response creates on file. The QCD Smearing creates two files which are then hadded together in the background.

File: python/processors/ahadd.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
```
Comment (zhenbinwu) [inline python/processors/ahadd.py]: How is this script comparing to the haddnano below? It seems both are just for hadd.

**Comment by mkilpatr** [comment]
Yes but it needed to be in the directory of the running process. I’ll
change it to the absolute path of the original one from nanoaodtools.

**Comment by mkilpatr** [comment]
Readme instructions

**Comment by mkilpatr** [comment]
Made the qcd smearing able to run over condor and created instructions as readme.txt

**Pull request merged**